app/dashboard/templates/dashboard/hackathon/ethhack_2019.html

```diff
@@ -0,0 +1,84 @@
+      <a class="btn font-weight-bold pbtn" style="background-color: #f9006c; border-radius: 30px; width: 12rem; font-size: 1rem; margin-top: 2rem; margin-bottom: 3rem">Register</a>
+    </section>
+
+    <section class="container-fluid header" style="overflow: visible; background-image: url(/static/v2/images/hackathon/ethback2.png);">
+      <h2 class="font-weight-bolder part" style="color: white; padding-top: 3rem; font-size: 1.2rem; text-decoration: underline; text-underline-position: under;">Fund a Prize Bounty</h2>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-8 text-left">
+            <h2 class="pdesc2" style="color: white; padding-top: 1.25rem; padding-right: 3rem; font-size: 0.6rem">Have an epic challenge you want to see the community tackle? Want to see how bounty hunters from around the world use your tools? Reach out to us to sponsor and post a prize bounty!</h2>
+            <a class="btn font-weight-bold pbtn" style="background-color: #f9006c; border-radius: 30px; width: 12rem; font-size: 1rem; margin-top: 2rem; margin-bottom: 3rem">Fund a Prize</a>
+          </div>
+          <div class="col-4">
+            <img class="img-fluid bot" style="width: 100%; padding-bottom: 5rem" src="{% static "v2/images/hackathon/bot.svg" %}" />
+        </div>
+      </div>
+    </section>
+
+    <section style="background: white">
+      <h2 class="font-weight-bolder part" style="color: black; padding-top: 3rem; font-size: 1.2rem; text-decoration: underline; text-underline-position: under">Join the Commmunity</h2>
+      <h2 class="pdesc" style="color: black; padding-top: 1.25rem; padding-right: 3rem; padding-left: 3rem; font-size: 0.6rem">Register by April 1st for updates on key dates, prizes, teams, and instructions on how to start bounty hunting.</h2>
+      <a class="btn font-weight-bold pbtn" style="background-color: #f9006c; border-radius: 30px; width: 12rem; font-size: 1rem; margin-top: 2rem; margin-bottom: 3rem">Join</a>
+    </section>
+
+    <section style="background: black">
+      <div class="row align-items-center bottomnav" style="padding: 1rem">
+        <div class="col-3">
+          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/logo.png" %}" />
+        </div>
+        <div class="col-3">
+          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/microsoft.svg" %}" />
+        </div>
+        <div class="col-3">
+          <img class="img-fluid align-middle" style="width: 100%" src="{% static "v2/images/hackathon/consensys.png" %}" />
+        </div>
+        <div class="col-3">
+          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/ethmain.svg" %}" />
+        </div>
+      </div>
+    </section>
+
+    <script src="{% static "v2/js/bootstrap.min.js" %}"></script>
```

Comment:
if you are using bootstrap.js you need to also call pooper.js or is going to fail

Reply:
fixed